quadrat/sass/blocks/_post-title.scss

```diff
@@ -0,0 +1,7 @@
+.wp-block-post-title {
+	margin-bottom: var(--wp--custom--gap--vertical);
+
+	&:empty {
```

Comment:
IMO this shouldn't be needed — if the element is empty, why render it? But it works for now.